docs/walkthrough/diffs.rst

```diff
@@ -0,0 +1,114 @@
+==================
+Diffing the fields
+==================
+
+Previously (:doc:`updates`), we have made the PVC size updated
```

Comment:
"we have set the size of the PVC to be updated every time..." ..." i.e. introduced a cascaded update"